examples/base-cra-system-ts/src/App.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { Box, Stack, styled } from '@mui/system';
```

Comment:
Let's use the way of importing dependencies we recommend:
```suggestion
import Box from '@mui/system/Box';
import Stack from '@mui/system/Stack';
import styled from '@mui/system/styled';
```

+ other examples